### DISCUSSION TITLE
code not  work properly

### DISCUSSION THREAD

**Discussion by pgreen24** [community/community#68316] [Programming Help]
### Select Topic Area

Bug

### Feature Area

Projects

### Details
i created a  chatboot that was not work ,and i also not find out where is the exactly problem in my code

### Expected Result

When the ask question about something (which i  write in the code ) that will be answer 

### Actual Result
just show the "chat boot" nothing show me anything i write in the code , 
i tried to findout where is the actual problem but i did'nt 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by itsdotnet** [comment]
I think using ChatGPT API to create is very easy way

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @pgreen24 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
